### PR TITLE
Updated keyboard layout to new standard.

### PR DESCRIPTION
**Pull request by jurassic73**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

Updated keymap to correct standard. Added matrix values to info.json. Worked with noroadsleft to update.

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/handwired/split89/info.json [hunk 1]
```diff
@@ -10,7 +10,97 @@
...
+            "layout": [
+                    {"label":"Esc",  "matrix": [0, 4],  "x":0 ,  "y":0},
+                    {"label":"F1" ,  "matrix": [0, 5],  "x":1 ,  "y":0},
+                    {"label":"F2" ,  "matrix": [0, 6],  "x":2 ,  "y":0},
+                    {"label":"F3" ,  "matrix": [0, 7],  "x":3 ,  "y":0},
+                    {"label":"F4" ,  "matrix": [0, 8],  "x":4 ,  "y":0},
+                    {"label":"F5" ,  "matrix": [0, 9],  "x":5 ,  "y":0},
```
Comment (noroadsleft) [inline keyboards/handwired/split89/info.json:19]:
```suggestion
                    {"label":"F1" ,  "matrix": [0, 5],  "x":2 ,  "y":0},
                    {"label":"F2" ,  "matrix": [0, 6],  "x":3 ,  "y":0},
                    {"label":"F3" ,  "matrix": [0, 7],  "x":4 ,  "y":0},
                    {"label":"F4" ,  "matrix": [0, 8],  "x":5 ,  "y":0},
                    {"label":"F5" ,  "matrix": [0, 9],  "x":6 ,  "y":0},
```

File: keyboards/handwired/split89/info.json [hunk 2]
```diff
@@ -10,7 +10,97 @@
...
+                    {"label":"Insert" ,  "matrix": [7, 7], "x":18 ,  "y":1.5},
+                    {"label":"Home" ,  "matrix": [7, 8], "x":19 ,  "y":1.5},
+                    {"label":"PgUp" ,  "matrix": [7, 9], "x":20 ,  "y":1.5},
+                    {"label":"Tab" ,  "matrix": [3, 4], "x":0 ,  "y":2.5 ,  "w":1.5},
+                    {"label":"Q" ,  "matrix": [2, 5], "x":1.5 ,  "y":2.5},
+                    {"label":"W" ,  "matrix": [2, 6], "x":2.5 ,  "y":2.5},
+                    {"label":"E" ,  "matrix": [2, 7], "x":3.5 ,  "y":2.5},
+                    {"label":"R" ,  "matrix": [2, 8], "x":4.5 ,  "y":2.5},
+                    {"label":"T" ,  "matrix": [2, 9], "x":5.5 ,  "y":2.5},
+                    {"label":"Y" ,  "matrix": [9, 0], "x":9.25 ,  "y":3.5},
```
Comment (noroadsleft) [inline keyboards/handwired/split89/info.json:53]:
```suggestion
                    {"label":"Y" ,  "matrix": [9, 0], "x":9.25 ,  "y":2.5},
```

File: keyboards/handwired/split89/info.json [hunk 3]
```diff
@@ -10,7 +10,97 @@
...
+                    {"label":"Delete" ,  "matrix": [8, 7], "x":18 ,  "y":2.5},
+                    {"label":"End" ,  "matrix": [8, 8], "x":19 ,  "y":2.5},
+                    {"label":"PgDn" ,  "matrix": [8, 9], "x":20 ,  "y":2.5},
+                    {"label":"Caps Lock" ,  "matrix": [4, 4], "x":0 ,  "y":3.5 ,  "w":1.75},
+                    {"label":"A" ,  "matrix": [3, 5], "x":1.75 ,  "y":3.5},
+                    {"label":"S" ,  "matrix": [3, 6], "x":2.75 ,  "y":3.5},
+                    {"label":"D" ,  "matrix": [3, 7], "x":3.75 ,  "y":3.5},
+                    {"label":"F" ,  "matrix": [3, 8], "x":4.75 ,  "y":3.5},
+                    {"label":"G" ,  "matrix": [3, 9], "x":5.75 ,  "y":3.5},
+                    {"label":"H" ,  "matrix": [10, 0], "x":9.5 ,  "y":4.5},
```
Comment (noroadsleft) [inline keyboards/handwired/split89/info.json:70]:
```suggestion
                    {"label":"H" ,  "matrix": [10, 0], "x":9.5 ,  "y":3.5},
```

File: keyboards/handwired/split89/info.json [hunk 4]
```diff
@@ -10,7 +10,97 @@
...
+                    {"label":":" ,  "matrix": [9, 4], "x":13.5 ,  "y":3.5},
+                    {"label":"\"" ,  "matrix": [9, 5], "x":14.5 ,  "y":3.5},
+                    {"label":"Enter" ,  "matrix": [9, 6], "x":15.5 ,  "y":3.5 ,  "w":2.25},
+                    {"label":"Shift" ,  "matrix": [5, 4], "x":0 ,  "y":4.5 ,  "w":2.25},
+                    {"label":"Z" ,  "matrix": [4, 5], "x":2.25 ,  "y":4.5},
+                    {"label":"X" ,  "matrix": [4, 6], "x":3.25 ,  "y":4.5},
+                    {"label":"C" ,  "matrix": [4, 7], "x":4.25 ,  "y":4.5},
+                    {"label":"V" ,  "matrix": [4, 8], "x":5.25 ,  "y":4.5},
+                    {"label":"B" ,  "matrix": [4, 9], "x":6.25 ,  "y":4.5},
+                    {"label":"N" ,  "matrix": [11, 0], "x":10 ,  "y":5.5},
```
Comment (noroadsleft) [inline keyboards/handwired/split89/info.json:83]:
```suggestion
                    {"label":"N" ,  "matrix": [11, 0], "x":10 ,  "y":4.5},
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]
Fixes some visual position issues with the QMK Configurator implementation:

image

File: keyboards/handwired/split89/info.json (hunk 4 above)
Comment (jurassic73) [reply keyboards/handwired/split89/info.json:83]: This has an image of it for reference, if needed.

https://github.com/jurassic73/split89

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
I'm at a complete loss as to why the CI Build job keeps failing. :man_shrugging: Everything looks good IMO.

**Review by fauxpark — APPROVED** [review]

**Comment by jurassic73** [comment]
Thanks, all!!

**Pull request merged**